changelog.txt

```diff
@@ -1,6 +1,7 @@
 *** Changelog ***
 
 = 10.0.0 - xxxx-xx-xx =
+* Dev - Removes frontend code related to Payment Request Buttons in the checkout page
```

Comment:
I think we should flag this as an Update so we make it super-clear in the release notes that it is changing.
```suggestion
* Update - Removes frontend code related to Payment Request Buttons in the checkout page
```